flickr_download/flick_download.py

```diff
@@ -113,14 +113,21 @@ def download_set(set_id, get_filename, size_label=None):
 
     for photo in photos:
         fname = get_full_path(dirname, get_filename(pset, photo, suffix))
+        if 'video' in photo.getInfo():
+            fname = fname[:-3] + 'mp4'
```

Comment:
this will break if the filename suffix changes, and since we are constructing it ourselves we should probably do it properly :) So I suggest we stop adding `.jpg` in all the file handler functions and then in the `if` statement here for video we either append `.jpg` or `.mp4`